inst/odin/lancelot.R

```diff
@@ -1665,10 +1665,43 @@ update(general_tot) <- new_general_tot
 initial(hosp_tot) <- 0
 update(hosp_tot) <- new_ICU_tot + new_general_tot
 
+## cumulative deaths in hospital by age
 initial(D_hosp_tot) <- 0
 delta_D_hosp_tot <- sum(delta_D_hosp)
 update(D_hosp_tot) <- D_hosp_tot + delta_D_hosp_tot
 
+initial(D_hosp_tot_0_49) <- 0
+delta_D_hosp_tot_0_49 <- sum(delta_D_hosp[1:10]) + delta_D_hosp[18] * 3 / 8
+update(D_hosp_tot_0_49) <- D_hosp_tot_0_49 + delta_D_hosp_tot_0_49
+
+initial(D_hosp_tot_50_54) <- 0
+delta_D_hosp_tot_50_54 <- delta_D_hosp[11] + delta_D_hosp[18] * 1 / 8
+update(D_hosp_tot_50_54) <- D_hosp_tot_50_54 + delta_D_hosp_tot_50_54
+
+initial(D_hosp_tot_55_59) <- 0
+delta_D_hosp_tot_55_59 <- delta_D_hosp[12] + delta_D_hosp[18] * 2 / 8
+update(D_hosp_tot_55_59) <- D_hosp_tot_55_59 + delta_D_hosp_tot_55_59
+
+initial(D_hosp_tot_60_64) <- 0
+delta_D_hosp_tot_60_64 <- delta_D_hosp[13] + delta_D_hosp[18] * 2 / 8
+update(D_hosp_tot_60_64) <- D_hosp_tot_60_64 + delta_D_hosp_tot_60_64
+
+initial(D_hosp_tot_65_69) <- 0
+delta_D_hosp_tot_65_69 <- delta_D_hosp[14] + delta_D_hosp[19] * 1 / 8
+update(D_hosp_tot_65_69) <- D_hosp_tot_65_69 + delta_D_hosp_tot_65_69
+
+initial(D_hosp_tot_70_74) <- 0
+delta_D_hosp_tot_70_74 <- delta_D_hosp[15] + delta_D_hosp[19] * 1 / 8
+update(D_hosp_tot_70_74) <- D_hosp_tot_70_74 + delta_D_hosp_tot_70_74
+
+initial(D_hosp_tot_75_79) <- 0
+delta_D_hosp_tot_75_79 <- delta_D_hosp[16] + delta_D_hosp[19] * 1 / 8
+update(D_hosp_tot_75_79) <- D_hosp_tot_75_79 + delta_D_hosp_tot_75_79
+
+initial(D_hosp_tot_80_plus) <- 0
+delta_D_hosp_tot_80_plus <- delta_D_hosp[17] + delta_D_hosp[19] * 5 / 8
```

Comment:
Note that while you have split care home deaths (1/8, 1/8, 1/8, 5/8), CHR are drawn (0.05, 0.05, 0.15, 0.75).

Reply:
(And if anything, ages of CHR deaths are likely to skew higher than age of CHR population)

Reply:
Done